.github/pull_request_template.md

```diff
@@ -15,6 +15,7 @@
 - [ ] My code follows the code style of this project
 - [ ] I have added tests to cover my changes
 - [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
+- [ ] I have added metrics for my changes
```

Comment:
not all changes need metrics

but we also have similar wording about tests

Reply:
Added "if required" for further clarification